components/programs/ClassInstanceData.tsx

```diff
@@ -333,34 +419,89 @@ export const SummerDates: ClassDates = {
   demoDay: Date.parse("2020-07-25T13:00:00-07:00"),
 };
 
-function summerClass(
-  course: Course,
-  campus: Campus,
-  formAssemblyId: string
-): ClassInstance {
+export const FallDates: ClassDates = {
+  registrationDeadline: Date.parse("2020-08-23T20:00:00-07:00"),
+  interview: Date.parse("2020-09-02T12:00:00-07:00"),
+  demoDay: Date.parse("2020-12-12T13:00:00-07:00"),
+};
+
+// function summerClass(
+//   course: Course,
+//   campus: Campus,
+//   formAssemblyId: string
+// ): ClassInstance {
+//   return {
+//     type: "class",
+//     course,
+//     campus,
+//     classDates: SummerDates,
+//     meets: (
+//       <>
+//         Monday, Wednesday, Friday{" "}
+//         <Box component="span" display="inline-block">
+//           10:30am - 2:30pm
+//         </Box>
+//       </>
+//     ),
+//     startDate: "June 15th",
+//     endDate: "July 25th",
+//     signupUrl: `https://www.tfaforms.com/4804494?tfa_2013=${formAssemblyId}`,
+//   };
+// }
+
+function fallClassMW(course: Course, campus: Campus): ClassInstance {
```

Comment:
The `formAssemblyId` stuff works by inspecting the form to find the id of the selector and the values for each of the courses

<img width="1529" alt="Screen Shot 2020-07-25 at 9 44 37 AM" src="https://user-images.githubusercontent.com/26596/88461785-76e0a100-ce5b-11ea-8ac5-9710df46423d.png">